etc/helm/pachyderm/templates/pachd/deployment.yaml

```diff
@@ -236,8 +236,4 @@ spec:
         secret:
           secretName: {{ required "If pachd.tls.enabled, you must set pachd.tls.secretName" .Values.pachd.tls.secretName | quote }}
       {{- end }}
-      initContainers:
-      - name: init-etcd
-        image: busybox:1.28
-        command: ['sh', '-c', "until wget -O- http://etcd:2379/version > /dev/null; do echo waiting for etcd; sleep 2; done"]      
```

Comment:
look like we already have a 3 minute wait in pachd to connect to etcd that most operations block on, any idea if anything else relied on this?

Reply:
where do we set the 3 minute wait?

Reply:
https://github.com/pachyderm/pachyderm/blob/425ef6c1f55f030fa26d923ef788dcce4f82e80b/src/internal/serviceenv/service_env.go#L246